pages/api/hello.js

```diff
@@ -1,5 +1,10 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
 
-export default function handler(req, res) {
-  res.status(200).json({ name: 'John Doe' })
+export default async function handler(req, res) {
+  const response = await fetch(
+    "http://api.aviationstack.com/v1/flights?access_key=eef146cc9441c581c88b1ba9363a5d4b"
+  );
+
+  const data = await response.json();
+  res.status(200).json({ data: data });
```

Comment:
I think at this point there is no need to fetch in the api folder. 
For now it is reasonable to fetch in the frontend only, as some things are repeating themselves (for example response.json();  and data.data.data.

Reply:
Ok. It was suggested to put that function in the api folder from our coach. But I will change it!